code/go/0chain.net/blobbercore/challenge/challenge.go

```diff
@@ -100,19 +100,17 @@ func saveNewChallenge(c *ChallengeEntity, ctx context.Context) {
 	}
 
 	c.Status = Accepted
-	c.CreatedAt = common.ToTime(c.Created)
-	c.UpdatedAt = c.CreatedAt
 
 	logging.Logger.Info("[challenge]add: ",
 		zap.String("challenge_id", c.ChallengeID),
-		zap.Time("created", c.CreatedAt))
+		zap.Time("created", common.ToTime(c.CreatedAt)))
 
 	if err := db.Transaction(func(tx *gorm.DB) error {
 		return c.SaveWith(tx)
 	}); err != nil {
 		logging.Logger.Error("[challenge]add: ",
 			zap.String("challenge_id", c.ChallengeID),
-			zap.Time("created", c.CreatedAt),
+			zap.Time("created", common.ToTime(c.CreatedAt)),
```

Comment:
We'd better call `common.ToTime()`  on time, and use the result to avoid redundant time conversions.

Reply:
Done